### PR TITLE
NXBT-3188: use staging namespace for validation

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -70,12 +75,13 @@ pipeline {
...
               # upgrade Jenkins X platform
-              jx upgrade platform --namespace=webui \
+              jx upgrade platform --namespace=${NAMESPACE} \
                 --version 2.0.1547 \
                 --local-cloud-environment \
                 --always-upgrade \
                 --cleanup-temp-files=true \
-                --batch-mode
+                --batch-mode \
+                --verbose
```
Comment (ataillefer) [inline Jenkinsfile]: you might not want --verbose

**Comment by ataillefer** [comment]
Don't you want to specify the namespace when running `jx step credential -s jenkins-docker-cfg`?
Or do you assume that the secret has been duplicated from the webui to webui-staging namespace?

**Comment by ataillefer** [comment]
Noticed you don't scale the jenkins deployment with 0 then 1 replicas to restart it, as done for the platform team. Is `jx upgrade platform` enough?

**Comment by nelsonsilva** [comment]
> Don't you want to specify the namespace when running `jx step credential -s jenkins-docker-cfg`?
> Or do you assume that the secret has been duplicated from the webui to webui-staging namespace?

I want to get the secret from the current (webui) namespace (haven't imported or set up replication on webui-staging yet)

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**